### ISSUE TITLE
generateFilePaths excludes files with absolute path

### ISSUE THREAD

**Issue by felangel** [Workiva/dart_codemod#31]
`generateFilePaths` excludes valid file paths because when an absolute path is passed, `filePath` is initialized to a relative path

```dart
@override
Iterable<String> generateFilePaths() sync* {
  final dir = Directory(dirPath);
  for (final fse
      in dir.listSync(followLinks: followLinks, recursive: recursive)) {
    if (fse is! File) {
      continue;
    }
    final filePath = p.relative(fse.absolute.path); // convert to relative path
    if (!pathLooksLikeCode(filePath)) {...}
    ...
}
```

which means that `pathLooksLikeCode` will always return false because the path starts with `.`.

```dart
bool pathLooksLikeCode(String filePath) =>
    !filePath.contains('/.') &&
    !(filePath.startsWith('.') && !filePath.startsWith('./')) &&
    !filePath.startsWith('build/');
```

In my opinion this is unexpected behavior I would propose removing that condition in order to allow for absolute paths. Thoughts? Thanks!

**Comment by evanweible-wf** [comment]
I totally agree @felangel, we've discussed this on our team and we'd like to change how this works. For example, there are some scenarios where we _do_ want to run a codemod on files in a `build/` dir, which doesn't work with the current `pathLooksLikeCode` here, either.

I've been thinking that the simplest approach would be to have `runInteractiveCodemod()` accept an `Iterable<File>` instead of a `FileQuery`. That way the author of a codemod has complete control over of the files on which it runs. The `glob` package makes it easy to get a list of files that match a certain pattern, and then we can still provide some useful helper functions if needed.

I'll work on getting a PR for this up.

**Comment by evanweible-wf** [comment]
Sorry for the delay @felangel, but this is now fixed in v0.2.0
